### ISSUE TITLE
About accuracy?

### ISSUE THREAD

**Issue by majia-yu** [auroua/InsightFace_TF#2]
Hi, I am so interested in your code. Can you please tell me if it can achieve the same accuracy as in the paper?

**Comment by auroua** [comment]
The author use lots of `batchnorm` in the model. I only have one `Titan Xp gpu`, the max `batchsize` I can set is about 20. This `batchsize` is too small, the model can't convergence correctly. I think  `batchsize` is crucial for the model to get good result.
I still working on to find some tricks to get a comparable result.
You can reference  the following link [issues#91](https://github.com/deepinsight/insightface/issues/91) and [issues#86](https://github.com/deepinsight/insightface/issues/86).

**Comment by xsr-ai** [comment]
@auroua thanks for your great work.
And why you don‘t use the insightface pretrained model? you can convet it from mxnet format to tensorflow format, then continue training, i think that will help to convergence training.

**Comment by auroua** [comment]
@xsr-ai It's a good idea. I am training the model with some modify now. Maybe I will try in future.

**Comment by xsr-ai** [comment]
@auroua i use cosine loss to train FR, but after many strep learning, i only reach 98.7% ACC， i don‘t know why，and the training dataset is clean MSCeleb，and embedding size is 128. so, auroua, can you tell me how you train to get 99.12% ACC? and what ACC you reach is far from the paper had released. also, cosine loss and arcinsight loss are different, but they are nature same.

**Comment by ShuoYang129** [comment]
@xsr-ai do you  have the clean list text file of "clean MSCeleb"  or just the bin file?

**Comment by HsuTzuJen** [comment]
Hi, I just achieve 0.9913333333333332 in iter 259000(batch size 128, MS1M,single GPU) with  "tensorflow/tensorflow/contrib/slim/python/slim/nets/resnet_v2.py".
I only change the network. 

And 0.9911666666666668 in iter 278000 (batch size 128, VGG2 ,single GPU)

**Comment by xsr-ai** [comment]
@ShuoYang129 yes，i have msceleb clean list file，and i only attain which per people picture lager than 50，btw，i  trained with inception resnet v1 under facenet framework which had modified and its loss function are cosine loss and softmax loss，anymore，face size is 160 x 160，embedding size is 128.

**Comment by auroua** [comment]
Model C got comparable results. Please test this model.

**Comment by billtiger** [comment]
33

**Comment by justein** [comment]
@auroua  the model C is so big
